src/index.ts

```diff
@@ -12,35 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// ** This file is automatically generated by gapic-generator-typescript. **
-// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** This file is automatically generated by synthtool. **
+// ** https://github.com/googleapis/synthtool **
 // ** All changes to this file may be overwritten. **
 
 import * as v1 from './v1';
 import * as v1beta2 from './v1beta2';
 import * as v1p1beta1 from './v1p1beta1';
 import * as v1p2beta1 from './v1p2beta1';
 import * as v1p3beta1 from './v1p3beta1';
+
 const VideoIntelligenceServiceClient = v1.VideoIntelligenceServiceClient;
-const StreamingVideoIntelligenceServiceClient =
-  v1p3beta1.StreamingVideoIntelligenceServiceClient;
+
 export {
   v1,
   v1beta2,
   v1p1beta1,
   v1p2beta1,
   v1p3beta1,
   VideoIntelligenceServiceClient,
-  StreamingVideoIntelligenceServiceClient,
 };
-// For compatibility with JavaScript libraries we need to provide this default export:
```

Comment:
@alexander-fenster @bcoe let me know if you think this comment should stay in the `src/index.ts` template. Thanks

Reply:
I think we can remove the comment.